kofta/public/locales/cs/translation.json

```
@@ -138,23 +139,23 @@
       "addSupport": "a pokusíme se podporu přidat."
     },
     "inviteButton": {
-      "invited": "pozván",
-      "inviteToRoom": "pozvat do místnosti"
+      "invited": "Pozván",
+      "inviteToRoom": "Pozvat do místnosti"
     },
     "micPermissionBanner": {
       "permissionDenied": "Nemáš povolený mikrofon! Zkus znovu načíst stránku.",
-      "dismiss": "zavřít",
-      "tryAgain": "zkusit znovu"
+      "dismiss": "Zavřít",
+      "tryAgain": "Zkusit znovu"
     },
     "keyboardShortcuts": {
-      "setKeybind": "nastavit klávesovou zkratku",
-      "listening": "posluchám",
-      "toggleMuteKeybind": "ztlumit mikrofon",
-      "togglePushToTalkKeybind": "Pust-To-Talk",
-      "toggleOverlayKeybind": "toggle overlay keybind"
+      "setKeybind": "Nastavit klávesovou zkratku",
+      "listening": "Poslouchám",
+      "toggleMuteKeybind": "Ztlumit mikrofon",
+      "togglePushToTalkKeybind": "Push-To-Talk (mluvit jen po zmáčknutí tlačítka)",
```

```suggestion
      "mluvit jen po zmáčknutí tlačítka",
``` 

Shouldn't this be changed to this?

---

It shouldn't, one is not sufficiently descriptive without the other.